src/react/databrowser/objects/ObjectDelete.tsx

```diff
@@ -142,26 +305,88 @@ const ObjectDelete = ({
   const dispatch: DispatchAPI<Action> = useDispatch();
 
   const [toggledFiles, setToggledFiles] = useState([...toggled]);
-  const hasLockedFiles = toggledFiles.some(
-    (file) => file.lockStatus === 'LOCKED',
-  );
+
   const totalSize = useMemo(() => fileSizer(toggledFiles), [toggledFiles]);
 
   const [confirmed, setConfirmed] = useState(false);
   const provided = useMemo(() => ({ confirmed, setConfirmed }), [confirmed]);
   const theme = useTheme();
-  const [isVersionDeletionConfirmed, setIsVersionDeletionConfirmed] =
-    useState(false);
+  const [isCheckboxToggled, setIsCheckboxToggled] = useState(false);
   const isCurrentSelectionPermanentlyDeleted = isPermanentDelete(
     toggledFiles,
     bucketInfo.isVersioning,
   );
-  const markersStr = maybePluralize(toggled.size, 'marker', 's', false);
-  const objectsStr = maybePluralize(toggled.size, 'object', 's', false);
-  const versionsStr = maybePluralize(toggled.size, 'version', 's', false);
-  const notificationText = !isCurrentSelectionPermanentlyDeleted
-    ? `Delete ${markersStr} will be added to the selected ${objectsStr}.`
-    : `The selected ${versionsStr} will be permanently removed from the Bucket.`;
+
+  const getProtectedDeletionMessage = (file: Record<string, any>) => {
```

Comment:
We have a type describing this object in `src/types/s3.ts`
```suggestion
  const getProtectedDeletionMessage = (s3Object: ObjectEntity) => {
```